front/style/pages/_committee.scss

```diff
@@ -24,6 +24,7 @@
     }
 
     &__name {
+        display: flex;
```

Comment:
it's useless?

Reply:
No , there are a probleme with the texte under the svg if I don t do that :'(